### DISCUSSION TITLE
[Announcement] Final exam

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs424-program-reasoning#266] [Announcements]
Here are the details of the exam you must be aware of.
Please read carefully and be ready for the exam.

* Time: Dec 11. (Mon) 9:00 - 10:40 (100 mins)
* Place: N1 111 (same place as the lecture room)
* Open book but no digital devices allowed
* Your answers must be written in Korean if you are a native Korean student; English otherwise
* No questions are allowed during the exam. If anything is unclear, come up with your own valid assumption and provide it with the answer.
* Scoring will be performed in `And` operation: For each problem. you will only get points if every part of your answer is correct. In other words, unnecessary parts of your answers may lead to the deduction of the scores. Thus, please keep your answers brief.

**Comment by 08kmc09** [comment]
I forgot to bring my pen.. can I borrow one during the exam?

**Comment by goodtaeeun** [comment]
i will give you one

**Comment by 08kmc09** [comment]
I really appreciate a lot!!
